modules/cnf-du-configuring-workload-partitioning.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-configuring-workload-partitioning_{context}"]
+
+= Configuring workload partitioning
+
+The following procedure outlines a high level, end to end workflow that deploys a cluster with workload partitioning enabled and pods that are correctly scheduled to run on the management CPU partition.
+
+. The user creates a machine config manifest to configure CRI-O to partition management workloads. The cpuset specified must match the reserved cpuset specified in the performance-addon-operator profile.
+
+. The user creates another machine config manifest to write a configuration file for kubelet to enable the same workload partition. The file is only readable by the kubelet.
+
+. The user runs the installer to create the standard manifests, adds their extra manifests from steps 1 and 2, then creates the cluster.
+
+. For pods and namespaces that are correctly annotated CPU request values are zeroed out and converted to `{workload-type}.workload.openshift.io/cores`. This modified resource allows the pods to be constrained to the restricted CPUs.
+
+. The SNO cluster comes up with management components constrained to a subset of available CPUs.
```

Comment:
nit: SNO might not be needed at all.  If you prefer it, please replace with "single node cluster"

nit: "comes up" is informal (and I'm unsure how that translates), consider "starts"

nit: The preceding bullets were specific about a "cpuset" and "restricted CPUs", so "a subset of available CPUs" seems less precise and less deterministic.  Consider replacing with "...constrained to the restricted CPUs."